src/patternfly/components/Page/page.scss

```diff
@@ -285,6 +295,40 @@
       display: block;
       visibility: visible;
     }
+
+    > .pf-c-button {
+      width: var(--pf-c-page__header-tools-group--m-icons--c-button--Width);
+      height: var(--pf-c-page__header-tools-group--m-icons--c-button--Height);
+      padding: unset;
+      background-color: var(--pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor);
+      border-radius: var(--pf-c-page__header-tools-group--m-icons--c-button--BorderRadius);
+
+      > i {
```

Comment:
is this necessary? if the icon isn't centered, could you use `display: flex; align-items: center; justify-content: center;` on the button?

Reply:
yep that helps a lot!